routesrv/redishandler.go

```diff
@@ -27,11 +28,17 @@ func (rh *RedisHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 	w.Header().Add("Content-Type", "application/json")
-	w.Write(rh.AddrUpdater())
+	address, err := rh.AddrUpdater()
+	if err != nil {
+		log.Errorf("Could not update address for redis handler %v", err)
```

Comment:
`Could not update address for redis handler %v` -> `Failed to update address for redis handler %v`